baseplate/lib/prometheus_metrics.py

```diff
@@ -36,3 +41,19 @@ def getHTTPSuccessLabel(httpStatusCode: int) -> str:
     The HTTP success label is "true" if the status code is 2xx or 3xx, "false" otherwise.
     """
     return str(200 <= httpStatusCode < 400).lower()
+
+
+def are_metrics_enabled(raw_config: Dict[str, str]) -> bool:
+    if "metrics.tagging" in raw_config or "metrics.namespace" in raw_config:
```

Comment:
I think this is a bit confusing, since you could say `metrics.namespace = "harold"` but `metrics.enabled = False`

So that we work towards getting rid of the older settings names, how about the logic as follows (default handled by `or False`). This way you can still explicitly disable it, and the new setting takes priority without affecting those that are currently using the legacy way:

```python
cfg = config.parse_config(raw_config, {})
if hasattr(cfg.metrics, 'enabled'):
  return cfg.metrics.enabled
return "metrics.tagging" in raw_config or "metrics.namespace" in raw_config or False
```